### PR TITLE
Ui & parser v 0.01

### PR DESCRIPTION
**Pull request by JustinnT**
Thankss if I dont reply to the comments can y'all pm me on tele? Thanks <3

### CODE REVIEW

File: src/main/java/seedu/duke/Duke.java [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
     private Ui ui;
     private Parser parser;
     private Storage storage;
 
     public Duke() {
+        indexNumber = 0;
         ui = new Ui();
         parser = new Parser();
         storage = new Storage();
     }
```
Comment (brandoncjh) [inline src/main/java/seedu/duke/Duke.java:23]: I will probably add a startup method here to load the saved files and anything else (eg intro message)

**Review by brandoncjh — COMMENTED** [review]
uwu

File: src/main/java/seedu/duke/enums/PatientFieldKeys.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * The only exception is indexNumber, as the indexNumber for each patient is not dependant on the individual,
+ * but rather the system (duke).
+ */
+public enum PatientFieldKeys {
+
+    NAME,
+    AGE,
+    ADDRESS;
+
```
Comment (brandoncjh) [inline src/main/java/seedu/duke/enums/PatientFieldKeys.java:17]: put in telephone number

Comment (JustinnT) [reply src/main/java/seedu/duke/enums/PatientFieldKeys.java:17]: Ok):

Comment (JustinnT) [reply src/main/java/seedu/duke/enums/PatientFieldKeys.java:17]: done

**Pull request merged**